public/app/percona/backup/components/AddBackupModal/AddBackupModal.types.ts

```diff
@@ -20,11 +20,11 @@ export interface SelectableService {
 
 export interface AddBackupFormProps {
   id: string;
-  service: SelectableValue<SelectableService>;
+  service: null | SelectableValue<SelectableService>;
```

Comment:
nit: I think it looks better to have `null` after the `|`, it reads better to me

Reply:
👍